### DISCUSSION TITLE
Mergify requesting new GitHub Actions permissions

### DISCUSSION THREAD

**Discussion by bpholt** [Mergifyio/mergify#5076] [Q&A]
As admin of a couple orgs, I received requests from our Mergify integrations for new GitHub permissions, specifically
> **Read-only** access to **Actions**

I haven't been able to find what this is for documented anywhere in Mergify's documentation or changelog. Is this legitimate? For what will this new permission be used?

**Comment by jd** [comment]
Hi @bpholt,

We're working on a couple of features right now to have tighter integrations with GitHub Actions:
- [CI Optimizer](https://mergify.com/ci-optimizer)
- Ability to read more fine-grained status from GHA workflows and enhance our conditions system.

And probably more to come. :)
